apis/v1alpha1/nginxproxy_types.go

```diff
@@ -203,3 +207,43 @@ const (
 	// kubebuilder:validation:Pattern=`^[\.a-zA-Z0-9:]*(\/([0-9]?[0-9]?[0-9]))$`
 	AddressTypeCIDR AddressType = "cidr"
 )
+
+// NginxLogging defines logging related settings for NGINX.
+type NginxLogging struct {
+	// Level defines the error log level.
```

Comment:
```suggestion
	// Level defines the error log level. Possible log levels listed in order of increasing severity are debug, info, notice, warn, error, crit, alert, and emerg. Setting a certain log level will cause all messages of the specified and more severe log levels to be logged. For example, the default level error will cause error, crit, alert, and emerg messages to be logged. https://nginx.org/en/docs/ngx_core_module.html#error_log
```

Reply:
Should the default error log level we set NGINX to stay as `info` or should we change it to match the comment in the nginx documentation to `error`. otherwise I can just adjust the documentation.

Reply:
Ugh, sorry, I originally removed the "default log level error" statement from the suggestion, but then GitHub lost my comment, and I had to re-do it. Let's keep our default the same and adjust the documentation.